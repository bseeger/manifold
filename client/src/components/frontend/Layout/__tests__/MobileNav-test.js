import React from 'react';
import renderer from 'react-test-renderer';
import MobileNav from '../MobileNav';

describe("Frontend.Layout.MobileNav component", () => {

  it('renders correctly', () => {
    const component = renderer.create(
      <MobileNav
        location={{}}
      />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});

