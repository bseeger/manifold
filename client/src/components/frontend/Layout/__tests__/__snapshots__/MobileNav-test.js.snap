exports[`Frontend.Layout.MobileNav component renders correctly 1`] = `
<nav
  className="footer-fixed">
  <ul
    className="text-nav">
    <li>
      <a
        className="active"
        onClick={[Function]}
        style={Object {}}>
        Projects
      </a>
    </li>
    <li>
      <a
        className=""
        onClick={[Function]}
        style={Object {}}>
        Following
      </a>
    </li>
  </ul>
</nav>
`;
